src/pages/Ticket/TicketListItem.tsx

```diff
@@ -0,0 +1,167 @@
+import { Box, Card, ListItem, Skeleton, Stack, Typography } from '@mui/material';
+import {
+  ButtonType,
+  GetTicket,
+  TicketStatus,
+  TicketTargetType,
+} from '../../types/TicketTypes';
+import { transitionLong } from '../../theme';
+import { useRef, useState } from 'react';
+import { useMaxLines } from '../../utils/hooks';
+import { useUserDetails } from '../../api/user';
+import UserInfo from '../User/UserInfo';
+import TicketButton from '../../components/TicketButton';
+import TypographyLink from '../../components/TypographyLink';
+import { ROUTES } from '../../const';
+import { translateTicketStatus, translateTicketTargetType } from '../../utils/utils';
+import { useCommentById } from '../../api/comment';
+import Comment from '../Video/Comment/Comment';
+interface TicketListItemProps {
+  ticket: GetTicket;
+  isAdmin: boolean;
+}
+
+function TicketListItem({ ticket, isAdmin }: TicketListItemProps) {
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const reasonRef = useRef<HTMLParagraphElement>(null);
+  const { isEllipsisActive, style: reasonMaxLinesStyle } = useMaxLines(1, reasonRef);
+  const { data: submitterDetails, isLoading: isSubmitterDetailsLoading } = useUserDetails(
+    ticket.submitterId
+  );
+  const {
+    data: commentDetails,
+    isLoading: isCommentDetailsLoading,
+    error: commentError,
+  } = useCommentById(
+    ticket.targetType === TicketTargetType.Comment ? ticket.targetId : undefined
+  );
+  const {
+    data: commentResponseDetails,
+    isLoading: isCommentResponseDetailsLoading,
+    error: commentResponseError,
+  } = useCommentById(
+    ticket.targetType === TicketTargetType.CommentResponse ? ticket.targetId : undefined
+  );
```

Comment:
a nie mozesz po prostu raz tego hooka wywołać? 

```suggestion
  const isComment = [TicketTargetType.Comment, TicketTargetType.CommentResponse].includes(ticket.targetType);

  const {
    data: commentDetails,
    isLoading: isCommentDetailsLoading,
    error: commentError,
  } = useCommentById(
    isComment ? ticket.targetId : undefined
  );
```

Reply:
a mogę

Reply:
jednak nie